rpc-server/src/main.rs

```diff
@@ -152,105 +363,19 @@ async fn main() -> anyhow::Result<()> {
-            &modules::transactions::methods::broadcast_tx_async,
-        )
-        .with_method_and_metrics(
-            "broadcast_tx_commit",
-            &modules::transactions::methods::broadcast_tx_commit,
-        )
-        .with_method_and_metrics("chunk", &modules::blocks::methods::chunk)
-        .with_method_and_metrics("gas_price", &modules::gas::methods::gas_price)
-        .with_method_and_metrics("health", &modules::network::methods::health)
-        .with_method_and_metrics(
-            "light_client_proof",
-            &modules::clients::methods::light_client_proof,
-        )
-        .with_method_and_metrics(
-            "next_light_client_block",
-            &modules::clients::methods::next_light_client_block,
-        )
-        .with_method_and_metrics("network_info", &modules::network::methods::network_info)
-        .with_method_and_metrics("send_tx", &modules::transactions::methods::send_tx)
-        .with_method_and_metrics("status", &modules::network::methods::status)
-        .with_method_and_metrics("tx", &modules::transactions::methods::tx)
-        .with_method_and_metrics("validators", &modules::network::methods::validators)
-        .with_method_and_metrics("client_config", &modules::network::methods::client_config)
-        .with_method_and_metrics(
-            "EXPERIMENTAL_changes",
-            &modules::blocks::methods::changes_in_block_by_type,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_changes_in_block",
-            &modules::blocks::methods::changes_in_block,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_genesis_config",
-            &modules::network::methods::genesis_config,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_light_client_proof",
-            &modules::clients::methods::light_client_proof,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_protocol_config",
-            &modules::network::methods::protocol_config,
-        )
-        .with_method_and_metrics("EXPERIMENTAL_receipt", &modules::receipts::methods::receipt)
-        .with_method_and_metrics(
-            "EXPERIMENTAL_tx_status",
-            &modules::transactions::methods::tx_status,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_validators_ordered",
-            &modules::network::methods::validators_ordered,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_maintenance_windows",
-            &modules::network::methods::maintenance_windows,
-        )
-        .with_method_and_metrics(
-            "EXPERIMENTAL_split_storage_info",
-            &modules::network::methods::split_storage_info,
-        )
-        .finish();
-
+    // TODO: decide what to do with metrics here
     // Insert all rpc methods to the hashmap after init the server
-    metrics::RPC_METHODS.insert(rpc.router.routers()).await;
+    // metrics::RPC_METHODS.insert(rpc.router.routers()).await;
```

Comment:
@khorolets Any ideas how to insert all methods now? I don't want to hardcode them here